src/realm/bplustree.cpp

```diff
@@ -104,6 +104,7 @@ class BPlusTreeInner : public BPlusTreeNode, private Array {
         return old_ref;
     }
     void ensure_offsets();
+    BPlusTreeLeaf* cache_leaf(MemRef mem, size_t ndx, size_t offset);
```

Comment:
This doesn't need to be moved?

Reply:
At some point it did. But I stepped back from that modification.